Source/Fuse.Reactive/ObserverMap.uno

```diff
@@ -54,14 +54,16 @@ namespace Fuse.Reactive
 		{
 			Detach();
 			_observable = obs;
-			_slave = slave;
```

Comment:
This bit is likely to conflict with https://github.com/fusetools/fuselibs-public/pull/487
That PR also fixes this issue, in the same way, but with other changes as well.  Maybe we can remove the changes to ObserverMap from this PR and get that other PR merged to 1.3, then merged to master first? Or just deal with conflict later?

Reply:
I think that sounds like a good approach. Please ping me when it's merged and I'll rebase.